documentation/backup.md

```diff
@@ -0,0 +1,49 @@
+# Backup Puppet Enterprise using the PEADM module
+
+## What is being backed up?
+
+By default the `peadm::backup` plan will backup the following items:
```

Comment:
By "items", do we mean "databases"? Is peadm::backup meant for _only_ databases?

Reply:
Although a large part of what it backsup is the databases it takes some other files and information from API calls

I don't know if we should highlight next to each item what is done

classification is an API call 
ca is certificate files via the puppet-backup script